test/definitions/amount_energy_from_mass_and_temperature_test.py

```diff
@@ -16,54 +16,52 @@ def test_args():
     m = units.Quantity('m')
     SI.set_quantity_dimension(m, units.mass)
     SI.set_quantity_scale_factor(m, 0.5 * units.kilogram)
-    t2 = units.Quantity('t2')
-    SI.set_quantity_dimension(t2, units.temperature)
-    SI.set_quantity_scale_factor(t2, 373 * units.kelvin)
     t1 = units.Quantity('t1')
     SI.set_quantity_dimension(t1, units.temperature)
-    SI.set_quantity_scale_factor(t1, 273 * units.kelvin)
-    Args = namedtuple('Args', ['C', 'm', 't2', 't1'])
+    SI.set_quantity_scale_factor(t1, 323 * units.kelvin)
+    t2 = units.Quantity('t2')
+    SI.set_quantity_dimension(t2, units.temperature)
+    SI.set_quantity_scale_factor(t2, 273 * units.kelvin)
```

Comment:
Does not correspond to description. You are supposed to heat the body.

Reply:
It's not clear what the discrepancy is, added the conversions.

Reply:
T1 is supposed to be initial temperature.

Reply:
corrected the parameter transfer order